.github/workflows/ci.yml

```diff
@@ -75,7 +72,7 @@ jobs:
       - name: Setup dart
         uses: dart-lang/setup-dart@v1
```

Comment:
dart-lang/setup-dart@v1.4

Reply:
Four places of this topic were used, all four places were changed